README.md

```diff
@@ -3,17 +3,19 @@
 
 ![build](https://github.com/twitter/compose-rules/actions/workflows/build.yaml/badge.svg) ![Latest version](https://img.shields.io/maven-central/v/com.twitter.compose.rules/common)
 
-A big challenge to face when a big team with a large codebase starts adopting Compose is that not everybody will start at the same page. This happened to us at Twitter.
+It can be challenging for bit teams to start adopting Compose, particularly because not everyone will start at same time or with same patterns. Twitter tried to ease the pain by creating a set of Compose Lint rules.
```

Comment:
typo:
"bit" = "big"

Reply:
typo "at same time" should be "at the same time"

"with same patterns" should be "with the same patterns"

Reply:
just caught the bit one, on the other one, btw you can do it as a suggestion so I can auto apply